code/_core/obj/item/fishing/rod.dm

```diff
@@ -27,6 +28,7 @@
 
 	var/next_tick = 0
 	var/nospam
+	var/compact = FALSE
```

Comment:
compact should be saved to the savefile. Check out the _magazine.dm file for how items are saved.

Reply:
most people are gonna store it inside tackle boxes which they have to be required to be compact = false for that to begin with, im not sure its important add it to the save file? correct me if im wrong

Reply:
I think burgerstation has a safety where if an item is too big to fit in something, it drops it when someone spawns. Not sure though but I rather be on the safe side.

Reply:
done